### PR TITLE
[TDD] Kata - String Calculator

### PR DESCRIPTION
**Pull request by kwong21**
@fengyuanyang , first pass for the StringCalculator

Important Checklist

[x] I have written unit test first, make it red(fail first).
[x] I have then written logic code, make it green(success)
[x] I have tried to refactor my code, make it understandable(document my code)

### CODE REVIEW

**Comment by kwong21** [comment]
Hey @fengyuanyang ,

I don't see the feedback you mentioned in the issue ticket - #74

**Comment by fengyuanyang** [comment]
Hi @kwong21 
Can you see my comments here ?

**Comment by kwong21** [comment]
> Hi @kwong21
> Can you see my comments here ?

hey,

Yes I can see this comment but don't see the PR comments you mentioned.

**Comment by kwong21** [comment]
Hey @fengyuanyang ,

I don't see any feedback here in the PR. Can you let me know whats needed to resolve this?

**Review by fengyuanyang — COMMENTED** [review]
please check commit here

File: src/main/java/com/ordestiny/tdd/kata/StringCalculator.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+		if (numbers.isEmpty()) {
+			return "0";
+		}
+
+		if (isInvalidLastCharacterIn(numbers)) {
+			throw new IllegalArgumentException("Number expected but found EOF");
+		}
+
+		double result = getSum(numbers, delimiter);
```
[the remark this answers is not in the document]
Comment (kwong21) [reply src/main/java/com/ordestiny/tdd/kata/StringCalculator.java:32]: I am already using a string[] for numbers. 

The string number gets split into a String[] in `getSum` on line 69.

I changed the sum method on line 80 to get the total.

File: src/test/java/com/ordestiny/tdd/kata/StringCalculatorTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class StringCalculatorTest {
+
+	private StringCalculator fixture;
+
+	@BeforeEach
+	public void setup() {
+		this.fixture = new StringCalculator();
+	}
```
[the remark this answers is not in the document]
Comment (kwong21) [reply src/test/java/com/ordestiny/tdd/kata/StringCalculatorTest.java]: This was changed to use the `@BeforeAll` annotation

**Comment by kwong21** [comment]
It just struck me - did you start a review on github but did not complete the review? If that is the case, I would not have seen your comments until that review was submitted.

Unfortunately, this didn't make it in for hacktober, but it was good practice nonetheless

**Comment by fengyuanyang** [comment]
I am really sorry about that .It loos great for me now.
I did not complete the review , sorry.

**Pull request merged**